src/main/java/org/apache/maven/plugins/changes/ChangesMojo.java

```diff
@@ -191,7 +191,7 @@
      *
      * @since 2.4
      */
-    @Parameter( defaultValue = "team-list.html" )
+    @Parameter( defaultValue = "team.html" )
     private String teamlist;
```

Comment:
You shouls also rename the field name to make it consistent.

Reply:
Done, the parameter is now "team".